frontend/src/components/maps/leaflet/InfoSlideOut/BuildingAttributes.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IBuilding } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import './InfoSlideOut.scss';
+import { formatMoney } from 'utils/numberFormatUtils';
+
+interface IBuildingAttributes {
+  /** the selected building information */
+  buildingInfo: IBuilding | null;
+}
+
+/**
+ * Displays Building specific information needed on the information slide out
+ * @param buildingInfo the selected parcel data
+ */
+export const BuildingAttributes: React.FC<IBuildingAttributes> = ({ buildingInfo }) => {
+  const formatAssessed = buildingInfo?.assessed
+    ? formatMoney(buildingInfo?.assessed)
+    : formatMoney(buildingInfo?.evaluations[0].value);
```

Comment:
for whatever reason the property detail sent from the SelectedPropertyMarker is in a different format so sometimes it gets the single assessed value and sometimes it doesn't

Reply:
I think if buildinginfo.assessed and buildingInfo.evaluations are both [] this will throw an arrayindexoutofbounds

Reply:
Okay, that makes a bit more sense thank you.